### ISSUE TITLE
Documentation & demo site down

### ISSUE THREAD

**Issue by cleverinx** [Fantomas42/django-blog-zinnia#568]
# Actual behavior
This site can’t be reached
docs.django-blog-zinnia.com’s server IP address could not be found.
DNS_PROBE_FINISHED_NXDOMAIN

# Expected behavior
Documentation area that works

# Steps to reproduce the issue

  1. Visit http://docs.django-blog-zinnia.com/en/latest/index.html
 

# Disclaimer

Before submitting an *issue* make sure you have:

 - [x] Read the guidelines for contributing.
 - [x] Checked for duplicate issues.
 - [x] Not a support request.

**Comment by 37dev** [comment]
Any updates on this?

**Comment by pyjavo** [comment]
Do you happen to know where can I find the documentation?

**Comment by maellacour** [comment]
Yes, as the site is down you can read them inside this repo under docs/

**Comment by pyjavo** [comment]
Thanks @Eyap53 
@cleverinx I think it will be better if the documentation can be moved to readthedocs and the demo website can be showcased in a videotutorial. That way it will be cheaper for the maintainer.

**Comment by elarroba** [comment]
@pyjavo @Eyap53 @cleverinx 
I've started #583 to start discussion on possible fork of this project for continued development.

**Comment by elarroba** [comment]
@pyjavo @Eyap53 @cleverinx 

I just created a fork of this project and updated the code to work with modern versions of Django (including 4.0+). 

Please visit [https://github.com/arrobalytics/django-blog-zinnia](https://github.com/arrobalytics/django-blog-zinnia) if you want to contribute. Don't forget to star the project to show your support. 

Visit the new site [https://www.djangoblogzinnia.com](https://www.djangoblogzinnia.com) for documentation, demo and other resources.
